texar/torch/core/layers.py

```diff
@@ -488,21 +487,19 @@ class instance, or a `dict` of hyperparameters of respective
                   "in_channels" not in hparams["kwargs"]):
                 raise ValueError("\"in_channels\" should be specified for "
                                  "\"torch.nn.{}\"".format(layer_class.__name__))
-            default_kwargs = _layer_class_to_default_kwargs_map.get(
-                layer_class, {})
+            default_kwargs: Dict[str, Any] = {}
             default_hparams = {"type": layer_type, "kwargs": default_kwargs}
             hparams = HParams(hparams, default_hparams)
 
-        # this case needs to be handled separately because
-        # :torch_nn:`Sequential`
+        # this case needs to be handled separately because nn.Sequential
         # does not accept kwargs
         if layer_type == "Sequential":
             names: List[str] = []
             layer = nn.Sequential()
             sub_hparams = hparams.kwargs.layers
             for hparam in sub_hparams:
                 sub_layer = get_layer(hparam)
-                name = utils.uniquify_str(sub_layer._get_name(), names)
+                name = utils.uniquify_str(sub_layer.__class__.__name__, names)
```

Comment:
`_get_name()` - is this one of the APIs removed in `pytorch 1.2`?

Reply:
It's still there, but they did not provide an annotation for that. Plus, this is supposed to be a private method, so I reverted to just using the class name.